gtk4/src/snapshot.rs

```diff
@@ -27,4 +27,19 @@ impl Snapshot {
     pub fn push_debug(&self, message: &str) {
         unsafe { ffi::gtk_snapshot_push_debug(self.to_glib_none().0, message.to_glib_none().0) }
     }
+
+    #[doc(alias = "gtk_snapshot_to_node")]
+    pub fn to_node(self) -> Option<gsk::RenderNode> {
```

Comment:
Should this really be `Option`s or `Result<T, glib::BoolError>`?

Reply:
What the `BoolError` should be? It's not necessarily an error when it returns `NULL`.

Reply:
Ah it's not? Then it should indeed be an `Option`